packages/mdc-textfield/foundation.js

```diff
@@ -152,6 +158,23 @@ class MDCTextFieldFoundation extends MDCFoundation {
     this.receivedUserInput_ = true;
   }
 
+  /**
+   * Handles validation attribute changes
+   */
+  handleValidationAttributeMutation(mutationsList) {
+    // whitelist based off of https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/HTML5/Constraint_validation
+    // under section: `Validation-related attributes`
+    const validationAttrWhitelist = [
```

Comment:
This can be hoisted to the top of the file above the class, rather than being defined every time we run this (and rename it `VALIDATION_ATTR_WHITELIST`)